kiwi/bootloader/install/grub2.py

```diff
@@ -42,12 +42,22 @@ def post_init(self, custom_args):
                 'root device node name required for grub2 installation'
             )
 
+        self.grub2_target = 'i386-pc'
+        self.grub2_install_dev = self.device
         self.root_mount = MountManager(
             custom_args['root_device']
         )
         self.boot_mount = MountManager(
             custom_args['boot_device']
         )
+        self.modules =' '.join(Defaults.get_grub_bios_modules())
+        self.grub2_install_args = ['--skip-fs-probe']
+        if 'prep_device' in custom_args:
```

Comment:
I don't meant checking the firmware, I meant checking for if prep_device exists in custom_args, if not raise. like it's done for boot_device

see the following code from post_init

```python
if not custom_args or 'boot_device' not in custom_args:
            raise KiwiBootLoaderGrubInstallError(
                'boot device node name required for grub2 installation'
            )
```